docs/section-6-contributing.md

```diff
@@ -48,48 +48,42 @@ This will create the `tree-sitter` CLI executable in the `target/release` folder
 Before you can run the tests, you need to clone some grammars that are used for testing:
 
 ```sh
-script/fetch-fixtures
+cargo xtask fetch-fixtures
 ```
 
 To test any changes you've made to the CLI, you can regenerate these parsers using your current CLI code:
 
 ```sh
-script/generate-fixtures
+cargo xtask generate-fixtures
 ```
 
 Then you can run the tests:
 
 ```sh
-script/test
+cargo xtask test
 ```
 
 Similarly, to test the WASM binding, you need to compile these parsers to WASM:
 
 ```sh
-script/generate-fixtures-wasm
-script/test-wasm
+cargo xtask generate-fixtures-wasm
```

Comment:
This isn't a command, I think you meant `cargo xtask generate-fixtures --wasm`?

Reply:
Fixed.